bibliopixel/main/all_pixel.py

```diff
@@ -109,3 +109,5 @@ def run(args, settings):
 
 def set_parser(parser):
     parser.set_defaults(run=run)
+    parser.add_argument('--hardware-id', default='1D50:60AB', type=str)
```

Comment:
Not sure what you mean by # 2
I'll add "help=" if you add it to your code ;)
From demo.py
def set_parser(parser):
    parser.set_defaults(run=run)
    parser.add_argument('name', nargs='?', default='')
    parser.add_argument('--width', default=16, type=int)
    parser.add_argument('--height', default=16, type=int)
    parser.add_argument('--depth', default=16, type=int)
    parser.add_argument('--simpixel', default=DEFAULT_SIMPIXEL_URL)